A32NX/html_ui/Pages/A32NX_Core/A32NX_FWC.js

```diff
@@ -65,6 +75,8 @@ class A32NX_FWC {
         this._updateButtons(_deltaTime);
         this._updateTakeoffMemo(_deltaTime);
         this._updateLandingMemo(_deltaTime);
+
+        this.altitudeWarning();
```

Comment:
```suggestion
        this._updatealtitudeWarning();
```
Just for naming consistency please.